app/uk/gov/hmrc/valuationofficeagencycontactfrontend/views/businessRatesSmartLinks.scala.html

```diff
@@ -34,6 +34,12 @@ <h1 class="heading-xlarge">@messages("checkBeforeYouStart.heading")</h1>
 
     <h2 class="heading-medium">@messages("checkBeforeYouStart.business_rates.subheading")</h2>
 
+    <p>
+        <a href=@messages("checkBeforeYouStart.business_rates.covid.url") target="_self">@messages("checkBeforeYouStart.business_rates.covid.link")</a>
```

Comment:
I think this doesn't render quotation marks in HTML. I don't know if we are able to do this in `twirl`.

Reply:
I am not sure what you mean. This is just a copy of the other links. It rendered without issue when I tested it.